string-handling/substrings/c-sharp/substring_demo.cs

```diff
@@ -0,0 +1,45 @@
+/*
+Isaac Computer Science
+Usage licensed under the Open Government Licence v3.0
+Note: This file is designed to be copied out and compiled on your machine. 
+In order for it to compile properly you need to ensure that the project name is the same as the "namespace" in this file. 
+To run this file you need to:
+1. Copy the contents
+2. Paste them into the C# IDE of your choice (Visual Studio, for example)
+3. Change the namespace to match your project (if neccesary)
+4. Compile the program
+5. Run the program
+*/
+
+using System;
+
+namespace IsaacCodeSamples
+{
+
+    class Substrings
+    {
+
+        // The Main method is the entry point for all C# programs
+        public static void Main(string[] args) {
+            ShowSubstrings();
+        }
+
+
+        // Demonstrates extracting substrings
+        public static void ShowSubstrings() {
+            string myString = "Computer Science";
```

Comment:
Maybe there's a reason for this, but it's a bit weird that this isn't a parameter

Reply:
It is - I will change it!